planb/cli.py

```diff
@@ -35,12 +36,32 @@ def validate_dc_suffix(ctx, param, value):
     return value
 
 
+def is_valid_filter(f: dict) -> bool:
+    return isinstance(f, dict) and \
+           isinstance(f.get('Name'), str) and \
+           isinstance(f.get('Values'), list)
+
+
+def validate_filters(ctx, param, value):
+    try:
+        filters = json.loads(value)
+    except Exception as e:
+        raise click.BadParameter("JSON parsing error: {}".format(e))
+
+    if not (isinstance(filters, list) and
+            all(is_valid_filter(f) for f in filters)):
+        raise click.BadParameter("has to match schema {}".format(filters_schema_doc))
```

Comment:
Start with uppercase ? Maybe s/has/Filter has/

Reply:
This is how the error message looks like: `Error: Invalid value for "--filters": has to match schema [{"Name":"string","Values":["string",...]},...]`

See also existing validations: https://github.com/zalando-stups/planb-cassandra/pull/239/files/a60962232bf808b79e85645f8ac79364b83af90c#diff-009a217c30d606342b4db9f6782bca0cL27